src/pages/RoomMemberDetailsPage.tsx

```diff
@@ -0,0 +1,124 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import Button from '@components/Button';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import MenuItem from '@components/MenuItem';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Report from '@libs/actions/Report';
+import type {RoomMembersNavigatorParamList} from '@libs/Navigation/types';
+import Navigation from '@navigation/Navigation';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type {PersonalDetails} from '@src/types/onyx';
+import NotFoundPage from './ErrorPage/NotFoundPage';
+import withReportOrNotFound from './home/report/withReportOrNotFound';
+import type {WithReportOrNotFoundProps} from './home/report/withReportOrNotFound';
+
+type RoomMemberDetailsPagePageProps = WithReportOrNotFoundProps & StackScreenProps<RoomMembersNavigatorParamList, typeof SCREENS.ROOM_MEMBERS.DETAILS>;
+
+function RoomMemberDetailsPage({report, route}: RoomMemberDetailsPagePageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const StyleUtils = useStyleUtils();
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
+    const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+
+    const [isRemoveMemberConfirmModalVisible, setIsRemoveMemberConfirmModalVisible] = React.useState(false);
+
+    const accountID = Number(route.params.accountID);
+    const backTo = ROUTES.ROOM_MEMBERS.getRoute(report?.reportID ?? '-1');
+
+    const member = report?.participants?.[accountID];
+    const details = personalDetails?.[accountID] ?? ({} as PersonalDetails);
+    const fallbackIcon = details.fallbackIcon ?? '';
+    const displayName = details.displayName ?? '';
```

Comment:
Here, we are directly using `details.displayName` to display the user's name, but it may contain the [SMS domain](https://github.com/Expensify/App/issues/53003) when the user has no name set and is logged in with a phone number. This could result in an extra '.sms' being displayed. We can use the [existing getDisplayNameOrDefault](https://github.com/Expensify/App/issues/53003#issuecomment-2495946266) to display names.